client/src/components/ElementEditor/Element.js

```diff
@@ -187,9 +187,9 @@ class Element extends Component {
     }
 
     if (type.inlineEditable && !loadingError) {
-      this.setState({
-        previewExpanded: !this.state.previewExpanded
-      });
+      this.setState((prevState) => ({
```

Comment:
As above

Reply:
See [this comment](https://github.com/silverstripe/silverstripe-elemental/pull/1070#discussion_r1253833874)